src/test/scala/chiseltest/backends/verilator/VerilatorCoverageTest.scala

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiseltest.backends.verilator
+
+import chiseltest.coverage.SimulatorCoverageTest
+import chiseltest.internal.VerilatorBackendAnnotation
+
+class VerilatorCoverageTest extends SimulatorCoverageTest("Verilator", VerilatorBackendAnnotation) {}
```

Comment:
Should this really be in its own file?
(or can you refactor it so that the treadle coverage tests are grammatically parallel, eg in its own file with the base class in a separate file?)

Reply:
Treadle is currently our default simulator, i.e. all tests are run with Treadle by default, so keeping the treadle test under the "default" package made sense to me. I wanted to move this test into the `verilator` package so that it would be caught by the `sbt testOnly ...` command I use to get all the Verilator tests.

If, in the future, we start changing tests so that we can run them with various backends, we could add a test option for that and keep them in a single file. For now, however, this is a one off test.